portal/server/controllers/utilisation-report-service/previous-reports/index.js

```diff
@@ -0,0 +1,49 @@
+const utilisationData = require('./data.json');
+
+const getMonthName = (monthNumber) => {
+  const date = new Date();
+  // offset by 1 as January = 1 in Database
+  date.setMonth(monthNumber - 1);
+  return date.toLocaleString('default', { month: 'long' });
+};
+
+const getUtilisationReportDownload = async (req, res) => {
+  // get data from the DB (currently using json) - will need to pass in bank id from user
+  try {
+    const { targetYear } = req.query;
+    const navItems = utilisationData.map((utilisation) => ({
```

Comment:
ditto here for making sure we always have an array in any error case coming from portal API, would prefer optional chaining to be safe in future if someone changes the API for example